app/flatpak-builtins-info-remote.c

```diff
@@ -207,7 +209,7 @@ flatpak_builtin_info_remote (int argc, char **argv, GCancellable *cancellable, G
       g_print ("%s%s%s %s\n", on, _("Parent:"), off, parent ? parent : "-");
       g_print ("%s%s%s %s\n", on, _("Download size:"), off, formatted_download_size);
       g_print ("%s%s%s %s\n", on, _("Installed size:"), off, formatted_installed_size);
-      if (strcmp (parts[0], "app") == 0)
+      if (strcmp (parts[0], "app") == 0 && metakey != NULL)
```

Comment:
Wondering if it would be more useful to always print the `Runtime:` line, but say it’s `unknown` if `metakey == NULL`. Alex?

Reply:
I dunno if that really helps anyone in practice. In the long run everything will get metadata.